lib/Conversion/ExportVerilog/ExportVerilog.cpp

```diff
@@ -4202,7 +4289,6 @@ void StmtEmitter::emitStatement(Operation *op) {
 
   emitOpError(op, "cannot emit this operation to Verilog");
   ps << "unknown MLIR operation " << PPExtString(op->getName().getStringRef());
-  setPendingNewline();
```

Comment:
I think we do want a newline after printing this, although it probably has not be tested as much as the rest (particularly the steps of extending EV to support new things), was/is there a problem with this?

Reply:
I don't think that I intended to keep this change in here, but it was causing a crash before I added the handler for the AggregateConstant op. With this line gone it just ends up hitting an assert.